src/base/datetime/calendar/header/test/changeEvent.test.ts

```diff
@@ -0,0 +1,95 @@
+import { expect, fixture } from "@open-wc/testing";
+import { BaseDateTimeCalendarHeader } from "../index";
+
+const props = {
+  month: 6,
+  year: 2021
+};
+
+describe("BaseDateTimeCalendarHeader", () => {
+  describe("kuc:calendar-header-change event", () => {
+    it("should be triggered when click the previous month button", async () => {
+      const container = new BaseDateTimeCalendarHeader(props);
+      container.addEventListener("kuc:calendar-header-change", (event: any) => {
+        expect(event.type).to.equal("kuc:calendar-header-change");
+        expect(event.detail.value).to.equal(`${props.year}-${props.month - 1}`);
```

Comment:
@dong0 
Please don't implementing logic in unit test.
NG:
expect(event.detail.value).to.equal(`${props.year}-${props.month - 1}`);

OK:
expect(event.detail.value).to.equal("2021-5");